packages/flutter_tools/lib/src/base/os.dart

```diff
@@ -2,6 +2,11 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// package:archive is used to provide zip/unzip only on Windows host machines. Since
+// Windows 8 and powershell 3, the native zip/unzip functionality has been exposed
+// and should be preferred over the dart implementation.
```

Comment:
Should we update the APIs herein to make that determination?

Reply:
This should probably be captured in a separate bug and not a comment.